nimble/host/include/host/ble_uuid.h

```diff
@@ -82,20 +82,26 @@ typedef union {
 
 #define BLE_UUID16_INIT(uuid16)         \
     {                                   \
-        .u.type = BLE_UUID_TYPE_16,     \
-        .value = (uuid16),              \
+        .u = {                          \
+            .type = BLE_UUID_TYPE_16,   \
+        },                              \
+        .value = (uuid16(,            	\
```

Comment:
almost there :)

while you change second `(` to `)` you can also remove tab before `\`

Reply:
Sorry didn´t see your message :)